core/parameters/res/runtime_configs/parameters.yaml

```diff
@@ -114,7 +114,7 @@ wasm_regular_op_cost: 3_856_371
 wasm_grow_mem_cost: 1
 wasm_base: 264_768_111
 wasm_contract_loading_base: 35_445_963
-wasm_contract_loading_bytes: 216_750
+wasm_contract_loading_bytes: 819_200
```

Comment:
worth double checking if modifying fees at genesis actually work at the current protocol versions and that these fees haven't been overriden later.

(you will notice that there are numbered yaml files each corresponding to changes on each protocol version; there is also a test that generates snapshots of the final computed parameter set)

Reply:
I manually looked at this files and didn't see an overriding of this numbers, but didn't find generated snapshot yet